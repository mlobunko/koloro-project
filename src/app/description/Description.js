import React from "react";
import PropTypes from "prop-types";

import {
  StyledStandardParagraph,
  StyledBoldParagraph,
  StyledH1,
  StyledItalicText
} from "./styles";

export const Description = ({ landscape }) => (
  <React.Fragment>
    <StyledH1 landscape={landscape}>
      <StyledItalicText>#КОЛОНИРУЙСЯ</StyledItalicText> - СТАНЬ ЧАСТЬЮ СЕМЬИ
      УСПЕШНЫХ КЛИЕНТОВ KOLORO
    </StyledH1>
    <StyledStandardParagraph landscape={landscape}>
      Брендинговое агентство KOLORO – создаем, позиционируем и продаем бренды.
      Индивидуальный подход к каждому проекту и восьмилетний опыт на рынке
      научили нас делать быстро, качественно и креативно!
    </StyledStandardParagraph>
    <StyledBoldParagraph landscape={landscape}>
      <div>ХОТИТЕ СОЗДАТЬ КОРОЛЕВСКИЙ БРЕНД?</div>
      <div>ТОГДА СКОРЕЕ ЗАПОЛНЯЙТЕ ФОРМУ?</div>
    </StyledBoldParagraph>
  </React.Fragment>
);

export default Description;

Description.propTypes = {
  landscape: PropTypes.bool.isRequired
};
