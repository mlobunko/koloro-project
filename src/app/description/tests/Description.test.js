import React from "react";
import { shallow } from "enzyme";

import { Description } from "../Description";

describe("render", () => {
  it("renders correctly", () => {
    const landscape = true;
    const wrapper = shallow(<Description landscape={landscape} />);
    expect(wrapper).toMatchSnapshot();
  });
});
