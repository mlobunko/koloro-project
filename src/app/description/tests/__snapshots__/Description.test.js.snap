// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render renders correctly 1`] = `
<Fragment>
  <StyledH1
    landscape={true}
  >
    <StyledItalicText>
      #КОЛОНИРУЙСЯ
    </StyledItalicText>
     - СТАНЬ ЧАСТЬЮ СЕМЬИ УСПЕШНЫХ КЛИЕНТОВ KOLORO
  </StyledH1>
  <StyledStandardParagraph
    landscape={true}
  >
    Брендинговое агентство KOLORO – создаем, позиционируем и продаем бренды. Индивидуальный подход к каждому проекту и восьмилетний опыт на рынке научили нас делать быстро, качественно и креативно!
  </StyledStandardParagraph>
  <StyledBoldParagraph
    landscape={true}
  >
    <div>
      ХОТИТЕ СОЗДАТЬ КОРОЛЕВСКИЙ БРЕНД?
    </div>
    <div>
      ТОГДА СКОРЕЕ ЗАПОЛНЯЙТЕ ФОРМУ?
    </div>
  </StyledBoldParagraph>
</Fragment>
`;
